### PR TITLE
[VEN-2377] [VEN-2378]: Restore the BinanceOracle for lisUSD and slisBNB

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN-2377

### CODE REVIEW

File: simulations/vip-248/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    it("Verify Symbol Override of lisUSD", async () => {
+      const symbol = await binanceOracle.symbols("lisUSD");
+      expect(symbol).equals("HAY");
+    });
+
+    it("Verify Symbol Override of slisBNB", async () => {
+      const symbol = await binanceOracle.symbols("slisBNB");
+      expect(symbol).equals("SnBNB");
+    });
```
Comment (chechu) [inline simulations/vip-248/bscmainnet.ts:67]: I would add checks getting the prices of the affected tokens, providing the underlying token address to the ResilientOracle, to reproduce the behaviour from the UI and the rest of the oracles users

File: vips/vip-248/bscmainnet.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ export const CRITICAL_TIMELOCK = "0x213c446ec11e45b15a6E29C1C1b402B8897f606d";
 export const FAST_TRACK_TIMELOCK = "0x555ba73dB1b006F3f2C7dB7126d6e4343aDBce02";
 export const ACM = "0x4788629abc6cfca10f9f969efdeaa1cf70c23555";
 export const ORACLE_GUARDIAN = "0x3a3284dC0FaFfb0b5F0d074c4C704D14326C98cF";
+export const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
```
Comment (chechu) [inline vips/vip-248/bscmainnet.ts:9]: I wouldn't declare this function here if we are not using it in this file

**Review by kkirka — APPROVED** [review]
LGTM

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/256?chainId=56

**Pull request merged**